VERSION

```diff
@@ -0,0 +1 @@
+0.1.0-beta
```

Comment:
Is there a chance we don't need to include a VERSION file for something that's in development? The Githash itself should be enough to tell us which commit this archer binary is built from, right? So version number for something in development does not seem too useful IMO.

I think it will be really nice if we simply use git tag as the source of truth for version number and when it does not exist, either don't print the version number at all or just use "development". By doing that, we can also get rid of the need to run a go program during build (i.e. generate-version.go)

What do you think?

Reply:
- Including version when in development: I personally think it's helpful to know what the latest version is when I'm in development, and it may also help future contributors. In the current ecs-cli, we only ever show the version number without indicating whether it is in development; the change [here](https://github.com/aws/amazon-ecs-cli/pull/928/files) only shows "development" which I find to be a little ambiguous.

- Using git tag as source of truth for version: This definitely makes sense from the point of view of building the command, but from the point of view of a human having a version file seems useful (i don't always check tags). Not sure if that's the convention though. Also, I wasn't totally sure how go modules does version lookups -- maybe it's tied to the tag as well.

I agree that having to run generate-version.go might be overkill for what we eventually want to do; we originally did this with the ecs-cli v1. But it works for now and it woudln't be that big a deal to remove this in the future.

Reply:
Why is this `VERSION` file necessary if we're using the `ldflags` linker flag to generate the version? It seems like this would need to be manually maintained in its current state unless I'm missing something. Can we automate it?

Reply:
@sonofachamp see the previous comment: 

`Using git tag as source of truth for version: This definitely makes sense from the point of view of building the command, but from the point of view of a human having a version file seems useful (i don't always check tags). Not sure if that's the convention though. Also, I wasn't totally sure how go modules does version lookups -- maybe it's tied to the tag as well.`

But if folks think we should just rely on tags etc that's fine.